.gitignore

```diff
@@ -23,4 +23,5 @@ paket-files
 TestResults
 *.log
 .vscode
-.idea
\ No newline at end of file
+.idea
+.ionide/
```

Comment:
I can revert this change if requested; it was necessary in my dev environment where I use the ionide plugin for VS Code.